src/containers/TransactionScenario/index.jsx

```diff
@@ -33,14 +37,18 @@ class TransactionScenario extends React.Component {
 		this.setState(_.cloneDeep(this.DEFAULT_STATE));
 	}
 
-	async submit() {
+	async submit(onChangeResetKeys, activeKeysData) {
```

Comment:
let's remove this parameters, cause this method is universal for all of our transactions, and should be independent and determined in all cases